metricbeat/module/elasticsearch/_meta/README.md

```diff
@@ -0,0 +1,62 @@
+# Elasticsearch Module
+
+- [About the elasticsearch metricbeat module](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-elasticsearch.html)
+- [List of exported fields](https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-elasticsearch.html) stored by this module generated from fields.yml files
+
+## Metricsets
+
+Metricbeat will call the following Elasticsearch API endpoints corresponding to each metricset.  The module stores a subset of the response in fields which can be found [here](https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-elasticsearch.html).  The Metricbeat exported fields generally follw the same paths as the elasticsearch API response, but may differ somewhat according to the [schema in each metricset](https://github.com/elastic/beats/blob/main/metricbeat/module/elasticsearch/node/data.go#L36).  They are [namespaced with `{module}.{metricset}`](https://github.com/elastic/beats/blob/main/metricbeat/module/elasticsearch/cluster_stats/data.go#L39).
+### ccr
+- `/_ccr/stats`
```

Comment:
I think this is a fine way to go for now, but I'd really like to see if we can get the specific paths into a unified code location like you can see in the kibana module https://github.com/elastic/beats/blob/main/metricbeat/module/kibana/kibana.go#L37-L39

That way we won't have to worry as much about keeping markdown and golang in sync over time.

Again, not something I'd block this PR on, just something I'd like us to keep in mind while we build out the module docs over time.

Reply:
Do you mean because having it in one place is easier to manually keep track of in terms of updating docs?  If we keep the path per metricset I suppose we could add a README to each metricset's `_meta` instead of this list in the top level `_meta`. Personally, I like the idea of the metricset path associated with the metricset since it's the only place it's being used.

It seems like other modules, aside from the `kibana` module, co-locate the path with the metricset so perhaps @sayden had a reason or could provide some guidance on whether we can move all the paths into the top level file.

Reply:
I might be wrong but I interpreted Mat's comment to be about duplicating the paths in Markdown vs somehow pointing to where they are defined in code so if they change the docs aren't out of date. But if we link to code location(s) I would worry what happens if the line or file changes where these are defined. 

Having a single api_paths constant file that lists the paths and exports them for use in code could be a good way to try to bridge this gap but I'm also ok with some duplication for now at the risk of staleness … docs are hard lol

Reply:
Yeah you got it @jasonrhodes - line numbers might move, but we can still standardize on a file then link from these docs to that file.

The docs for each path used can live with the constant that defines the path. Then if we add or remove any it becomes clear that the doc comments also need updating.

I'm not sure what we have going for `godoc` but constants should show up in those when they get generated. I was going to link to https://pkg.go.dev/github.com/elastic/beats/metricbeat#section-readme but sadly they don't allow our license.